### ISSUE TITLE
Live link not working

### ISSUE THREAD

**Issue by tomwayson** [Robert-W/esri-redux#6]
For me, the live link hangs like this:

![image](https://cloud.githubusercontent.com/assets/662944/22852245/ce1f0412-efec-11e6-8509-5899ded3062c.png)

With the following error in the console:
```
Refused to load the script 'https://www.arcgis.com/sharing/rest/portals/self?f=json&callback=dojo.io.script.jsonp_dojoIoScript1._jsonpCallback' because it violates the following Content Security Policy directive: "script-src 'self' js.arcgis.com 'unsafe-inline' 'unsafe-eval'".
```

Then after I've left the window and return I see:

![image](https://cloud.githubusercontent.com/assets/662944/22852255/f510a468-efec-11e6-81ad-907ffa992761.png)

With an additional error in the console:
```
s._attach @ init.js:396
init.js:356 dojo.io.script error k {message: "Timeout exceeded", response: Object, log: undefined, stack: "Error: Timeout exceeded↵    at k [as constructor] … at f (https://js.arcgis.com/4.0/init.js:178:388)"}
```

Then when I enter my credentials to view the map I see an additional error:
```
Uncaught TypeError: Cannot read property 'lods' of null
    at Object.getZoom (VectorTileLayerView2D.js:8)
    at Object.<anonymous> (VectorTileLayerView2D.js:5)
    at d (init.js:77)
    at h.isActive.c [as callback] (init.js:77)
    at h._dispatch (init.js:83)
    at c (init.js:90)
    at MutationObserver.observe.attributes (init.js:91)
```

This sounds vaguely like it might be related to [vector tile basemaps having been updated moved recently](https://blogs.esri.com/esri/arcgis/2016/05/13/updates-to-esri-vector-basemaps-released-beta-2/)?

Also couldn't hurt to bump to a more recent version of 4.x.

**Comment by Robert-W** [comment]
Hey @tomwayson, thanks for pointing this out.  I will update the CSP, JSAPI, and vector basemap urls later tonight. I have been meaning to come back to this for a little while now to do some more updates so I'll probably add those in as well. I want to get Webpack 2 in and have another branch that runs entirely in a docker container with the latest node version and other deps.

**Comment by Robert-W** [comment]
@tomwayson Thanks again for bringing this to my attention. Just pushed an update to gh-pages that is now working. I also made a few updates:

* Latest JSAPI, New Basemap option, added arcgis.com to the CSP
* Updated all npm dependencies (React, Redux, and Webpack)
* Updated all webpack config files to use v2 configurations
* Added yarn.lock (incase your using yarn)

I have only updated `master` so far, the other branches will be updated soon.
